test/parallel/test-https-server-connections-checking-leak.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+// Flags: --expose-gc
+
+// Check that creating a server without listening does not leak resources.
+
+const common = require('../common');
```

Comment:
```suggestion
require('../common');
```

Reply:
I can't here. I'm using on line 9 for the hasCrypto check.